utils/spacewalk-common-channels.ini

```
@@ -3179,3 +3179,51 @@ gpgkey_url = http://download.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8
 gpgkey_id = 2F86D6A1
 gpgkey_fingerprint = 94E2 79EB 8D8F 25B2 1810 ADF1 21EA 45AB 2F86 D6A1
 repo_url = http://mirrors.fedoraproject.org/mirrorlist?repo=epel-8&arch=%(arch)s
+
+[openeuler2203-os]
+archs	= x86_64, aarch64
+name	= Openeuler 22.03 OS %(arch)s
```

```suggestion
name	= openEuler 22.03 %(arch)s
```
Seems the official name is `openEuler`. Let's also use a neutral name here.